Makefile

```diff
@@ -11,18 +11,71 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-KEY_ID ?= _DEFINE_ME_
+all:
+	$(MAKE) go
+	$(MAKE) build
 
-all: go
+GO_FILE := go/metrics.pb.go
+PROTO_FILE := io/prometheus/client/metrics.proto
 
-SUFFIXES:
+# Need to be on a previous version that doesn't cause the updated WKT go_package values to be added.
+PROTOC_VERSION := 3.13.0
+# This has been around for a while.
+PROTOC_GEN_GO_VERSION := v1.3.5
```

Comment:
In theory we could use bingo too, to avoid so much things in Makefile to maintain: https://github.com/openproto/protoconfig/blob/main/go/Makefile#L52